README.md

```diff
@@ -21,136 +20,157 @@
-|----------------|-------|-------------|-------|---------------------------------------|
-|0x052f62696e2f  | /bin/ |raw binary   |binary | 0x00 |
-|0x042f62322f    | /b2/  |ascii base2  |binary | | 
-|0x052f6231362f  | /b16/ |ascii base16 |hex    | |
-|0x052f6233322f  | /b32/ |ascii base32 |       | | 
-|0x052f6235382f  | /b58/ |ascii base58 |       | |
-|0x052f6236342f  | /b64/ |ascii base64 |       | |
-|0x062f6a736f6e2f  |/json/ |             |json   | |
-|0x062f63626f722f  |/cbor/ |             |json   | |
-|0x062f62736f6e2f  |/bson/ |             |json   | |
-|0x072f626a736f6e2f|/bjson/|             |json   | |
-|0x082f75626a736f6e2f| /ubjson/|         |json   | |
-|0x182f6d756c7469636f6465632f | /multicodec/ | | multiformat | 0x40 |
-|0x162f6d756c7469686173682f   | /multihash/  | | multiformat | 0x41 |
-|0x162f6d756c7469616464722f   | /multiaddr/  | | multiformat | 0x42 |
-|0x0a2f70726f746f6275662f |/protobuf/ | Protocol Buffers |protobuf| |
-|0x072f6361706e702f       | /capnp/   | Cap-n-Proto      |protobuf| |
-|0x092f666c61746275662f   |/flatbuf/  | FlatBuffers      |protobuf| |
-|0x052f7461722f         |/tar/      |                 | archive | |
-|0x052f7a69702f         |/zip/      |                 | archive | |
-|0x052f706e672f         | /png/     |                 | archive | |
-|0x052f726c702f         | /rlp/     | recursive length prefix | ethereum |  0x60 |
-## The protocol path
-
-`multicodec` allows us to specify different protocols in a universal namespace, that way being able to recognize, multiplex, and embed them easily. We use the notion of a `path` instead of an `id` because it is meant to be a Unix-friendly URI.
-
-A good path name should be decipherable -- meaning that if some machine or developer -- who has no idea about your protocol -- encounters the path string, they should be able to look it up and resolve how to use it.
-
-An example of a good path name is:
 
-```
-/bittorrent.org/1.0
-```
-
-An example of a _great_ path name is:
-
-```
-/ipfs/Qmaa4Rw81a3a1VEx4LxB7HADUAXvZFhCoRdBzsMZyZmqHD/ipfs.protocol
-/http/w3id.org/ipfs/ipfs-1.1.0.json
+It is worth noting that multicodec-packed works very well in conjunction with [multihash](https://github.com/multiformats/multihash) and [multiaddr](https://github.com/multiformats/multiaddr), as you can prefix those values with a multicodec-packed to tell what they are.
+
+## Multicodec-Packed Protocol Tables
+
+Multicodecuses "protocol tables" to agree upon the mapping from one multicodec code (a single varint). These tables can be application specific, though -- like [with](https://github.com/multiformats/multihash) [other](https://github.com/multiformats/multibase) [multiformats](https://github.com/multiformats/multiaddr) -- we will keep a globally agreed upon table with common protocols and formats.
+
+## Multicodec table
+
+```csv
+codec,              description,              code
+
+miscelaneous
+bin,                raw binary,               0x00
```

Comment:
Binary could be 0x55 which is `0b10101010` and is also value of Ethernet frame preamble.